common/test/acceptance/pages/studio/overview.py

```diff
@@ -508,12 +508,6 @@ def change_section_name(self, new_name, parent_css=''):
         self.q(css='{} .section-name .save-button'.format(parent_css)).first.click()
         self.wait_for_ajax()
 
-    def click_release_date(self):
```

Comment:
Note to reviewers: when deleting overview.js, I noticed that this method (which has a class in it referred to by overview.js) can also be deleted.